webui/react/src/hooks/usePermissions.ts

```diff
@@ -230,6 +232,20 @@ const canViewGroups = ({ rbacReadPermission, rbacEnabled, user }: RbacOptsProps)
   return rbacReadPermission || (!!user && (rbacEnabled || user.isAdmin));
 };
 
+const canViewModelRegistry = ({
+  rbacReadPermission,
+  rbacEnabled,
+  user,
+  userAssignments,
+  userRoles,
+}: RbacOptsProps): boolean => {
+  const permitted = relevantPermissions(userAssignments, userRoles);
+  return (
+    rbacReadPermission ||
+    (!!user && (rbacEnabled ? permitted.has(V1PermissionType.VIEWMODELREGISTRY) : true))
```

Comment:
@keita-determined we can simplify this more. How about something such as 

```
rbacReadPermission || (!!user && (!rbacEnabled || permitted.has(V1PermissionType.VIEWMODELREGISTRY)))
```

Reply:
ok,  how about this? i also put a comment 

```ts
!rbacEnabled || rbacReadPermission || permitted.has(V1PermissionType.VIEWMODELREGISTRY)
```